### PR TITLE
04 get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by Mdmosley12**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -89,4 +89,35 @@ describe('app testing', () => {
...
+            })
+        })
+        test('200: All returned comments are sorted by date in descending order', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body: {comments} }) => {
+                expect(comments[0].comment_id).toBe(5);
+                expect(comments[10].comment_id).toBe(9);
+            })
```
Comment (MKD01) [inline __tests__/app.test.js]: Brill test, however i would recommend looking into the jest-sorted library for how to do this in a better way.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -89,4 +89,35 @@ describe('app testing', () => {
...
+        })
+    })
 })
```
Comment (MKD01) [inline __tests__/app.test.js:140]: Error handling can be added here for when an article id is given which doesnt exist i.e. /api/articles/999/comments as well as a test for when an invalid id is given i.e. /api/articles/Not-A-Valid-ID/comments

**Review by MKD01 — CHANGES_REQUESTED** [review]
Please do make those changes and tomorrow you can re-send this PR

File: __tests__/app.test.js (hunk 1 above)
Comment (Mdmosley12) [reply __tests__/app.test.js]: This has been implemented

File: __tests__/app.test.js (hunk 2 above)
Comment (Mdmosley12) [reply __tests__/app.test.js:140]: Both these tests have been added but also have been added on another branch

**Review by Rosem10 — APPROVED** [review]
Nice job Mitch!

**Pull request merged**